federatorai/README.md

```diff
@@ -0,0 +1,52 @@
+# Agent Check: federatorai
+
+## Overview
+
+This check monitors [ProphetStor Federator.ai][1].
+
+ProphetStor Federator.ai is an AI-based solution that helps enterprise manage, optimize, auto-scale resources for any applications on Kubernetes. Using advanced machine learning algorithms to predict application workload, Federator.ai scales the right amount of resources at the right time for optimized application performance.
+
+* AI-based workload prediction for Kafka or any applications
+* Resource recommendation based on workload prediction, application, Kubernetes and other related metrics
+* Automatic scaling of application containers through [Datadog Watermark Pod Autoscaler (WPA)][4]
+
+![federatorai dashboard][7]
+
+
+## Setup
+
+### Installation/Configuration
+
+ProphetStor Federator.ai for Kafka Acceleration
+
+Follow [Federator.ai Installation Guide][6] to install and configure [Federator.ai operator][2], [Datadog Kafka integration][3], [Datadog cluster agent][8] and [Datadog WPA][4].
```

Comment:
Can we provide the standard integration installation steps here (in Datadog docs) and refer to Federator to Propheststor.ai website for additional info? Here's an example for reference: https://docs.datadoghq.com/integrations/alcide/#configuration

We mostly want to make it easy to users to get going without navigating to multiple pages, where possible.

Also, link [1] is not live yet

Reply:
i have added installation steps to README.md.

Reply:
link [1] is reachable now.
[1] https://www.prophetstor.com/federator-ai-for-aiops/federator-ai-datadog-integration/